### DISCUSSION TITLE
Access Key & end point names is required for configuration part in SNOW

### DISCUSSION THREAD

**Discussion by Khansili123** [community/community#40906] [Actions]
### Select Topic Area

General

### Body

Access Key & endpoint names are required for the configuration part in SNOW, How to get the details as it is creating escalations?

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @Khansili123, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
